frontend/src/screens/learner/details.tsx

```diff
@@ -208,8 +215,16 @@ export const StudyDetailsPreview: FC<{
                     <StudyPart property="feedbackDescription" title="Feedback Available" icon="feedback" study={study} />
                     <MultiSession study={study} />
                     <Box margin={{ bottom: 'large' }} css={{ color: colors.text }}>{study.longDescription}</Box>
-                    {pi && <Researcher researcher={pi} />}
-                    {lead && <Researcher researcher={lead} />}
+                    {
```

Comment:
Good job! Just a little clean-up comment

Since the logic is getting a little more complex here (inside the render function), we can probably move this researcher section to it's own component. You can still keep the old <Researcher> component, but maybe add a <ResearcherSection> which you can pass the study into. 
  
Then you can extract all this logic out into that component and keep this one a little cleaner 😄 .

Once its in its own component, you can just do something like 
```
if (pi && lead && pi.id === lead.id) P
  return <Researcher researcher={pi} />
}

return (
  <>
    {pi && <Researcher researcher={pi} />}
    {lead && <Researcher researcher={lead} />}
  </>
)
```
something like that 🤔 just the general idea if that makes sense!

Reply:
Sounds good, makes sense! I implemented those changes in the commit below. Not sure if I did it correctly, so please let me know if there is anything else I need to fix. Thanks!